ocaml/idl/datamodel.ml

```diff
@@ -928,9 +928,11 @@ let _ =
   error Api_errors.pool_auth_disable_failed ["host";"message"]
     ~doc:"The pool failed to disable the external authentication of at least one host." ();
   error Api_errors.pool_auth_disable_failed_wrong_credentials ["host";"message"]
-    ~doc:"The pool failed to disable the external authentication of at least one host." ();
+    ~doc:"External authentication has been disabled with errors." ();
   error Api_errors.pool_auth_disable_failed_permission_denied ["host";"message"]
-    ~doc:"The pool failed to disable the external authentication of at least one host." ();
+    ~doc:"External authentication has been disabled with errors." ();
+  error Api_errors.pool_auth_disable_failed_invalid_account ["host";"message"]
+      ~doc:"External authentication has been disabled with errors." ();
```

Comment:
These three all have the same doc string, but the error code has more info that could be used in the text.

Also the indentation of this last line is slightly off.